gradle/release.gradle

```diff
@@ -1,3 +1,16 @@
+buildscript {
+    repositories {
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+    dependencies {
+        //so that PublicationsComparatorTask can be imported
+        //this entire 'buildscript' section will be removed when the entire release automation is moved to new repo 'mockito-release-tools'
```

Comment:
2 years ago it didn't work to use classes from external plugins in a separate gradle file without adding `buildscript` section in that file, even if a separate plugin was used. However, maybe it has been fixed in the meantime.